### ISSUE TITLE
Access Eleventy-provided filters in user configuration file.

### ISSUE THREAD

**Issue by rolandtoth** [11ty/eleventy#284]
For images I use this syntax:

```
<img src="{{ (cfg.urls.images + "/map.png") | url | cacheVersion }}" alt="">
```

This works fine but I would like to shorten this to something like this:

```
<img src="{{ "/map.png" | urlHelper("images") }}" alt="">
```

Then the urlHelper filter would add the "url" and "cacheVersion" filters. Is it possible somehow, or is any other recommended way?

**Comment by zachleat** [comment]
Ah, so you want access to Eleventy-provided filters in your configuration file. Hmm, I don’t think we currently provide a way to do this, no. It will need to go into the enhancement queue. Please upvote!

**Comment by zachleat** [comment]
This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

The enhancement backlog can be found here: https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+

Don’t forget to upvote the top comment with 👍!

**Comment by zachleat** [comment]
Shipping with 0.11.0

**Comment by zachleat** [comment]
Reopening for documentation add

**Comment by zachleat** [comment]
Configuration API method `getFilter`
